docker/builder/cmd.sh

```diff
@@ -59,6 +59,7 @@ for SUITE_ITERATION in $(seq 1 "${MAX_TEST_SUITE_RETRIES}"); do
       set +e
       daemon -U --name sauce_connect -- ${SC_BINARY} \
         -D *.ciscospark.com,*.wbx2.com,*.webex.com*,storage101.dfw1.clouddrive.com \
+        -t internal-testing-services.wbx2.com* \
```

Comment:
Is the star there for the port number? I'm reasonably confident this doesn't care about port numbers, just domain names.